ext/standaloneusers/sql/auto_uninstall.sql

```diff
@@ -15,7 +15,7 @@
 
 SET FOREIGN_KEY_CHECKS=0;
 
-DROP TABLE IF EXISTS `civicrm_user`;
+DROP TABLE IF EXISTS `civicrm_uf_match`;
 DROP TABLE IF EXISTS `civicrm_role`;
```

Comment:
Now that I think of it... there's really no reason to ever drop this table either.
Maybe we can just disable the whole file.
Or maybe don't worry about it for now.
Anyway, not a blocker to merge.